### ISSUE TITLE
Document URL syntax

### ISSUE THREAD

**Issue by saverkamp** [NYPL-discovery/discovery-models#27]
We need to document URL syntax for discovery URLs and add to Google Site.

**Comment by saverkamp** [comment]
We may need to change the syntax for electronic resource items generated from 856 fields. 
Currently: `http://dev-discovery.nypl.org/item/b19859287-i19859287-e`

Creating an item id based on the bib id might result in an item identifier collision or might be confusing to staff who may think it is a valid sierra id.

cc @nonword

**Comment by nonword** [comment]
The "-e" suffix should ensure no collision, but it is ugly so I'm open to suggestions. Would you prefer we generate an id like `http://dev-discovery.nypl.org/item/b19859287-1234` ? My thinking was that `i[bnum]-e` was guaranteed unique and reflective of what it is: an e- item generated from a bib. But it can be anything as long as it's unique.

**Comment by saverkamp** [comment]
Yes, it will definitely not conflict with other identifiers, so it'll work for now. What if we revisit it when we reconsider URL syntax for non-catalog entities--maybe shoot to have something in place for June beta?

I'm going to change this ticket to reflect the task of completing and documenting URL syntax for non-catalog entities, so we can keep this URL conversation in one place.

TO-DO
syntax for:
- electronic items (from 856)
- serial issues
- electronic resources (EBSCO)
- what else?
